src/MsgPack/Serialization/SerializationContext.cs

```diff
@@ -196,20 +196,20 @@ public SerializationCompatibilityOptions CompatibilityOptions
 			}
 		}
 
-		private readonly DictionarySerlaizationOptions _dictionarySerializationOptions;
+		private readonly DictionarySerializationOptions _dictionarySerializationOptions;
 
 		/// <summary>
 		///		Gets the dictionary(map) based serialization options.
 		/// </summary>
 		/// <value>
-		///		The <see cref="DictionarySerlaizationOptions"/> which stores dictionary(map) based serialization options. This value will not be <c>null</c>.
+		///		The <see cref="DictionarySerializationOptions"/> which stores dictionary(map) based serialization options. This value will not be <c>null</c>.
 		/// </value>
-		public DictionarySerlaizationOptions DictionarySerlaizationOptions
+		public DictionarySerializationOptions DictionarySerializationOptions
 		{
```

Comment:
```C#
[EditorBrowsable(EditorBrowsableState.Never)]
[Obsolete("Use DictionarySerializationOptions property instead.")]
public DictionarySerlaizationOptions DictionarySerlaizationOptions	
{
    get
    {
        return
            new DictionarySerlaizationOptions
            {
                KeyTransformer = this._dictionarySerializationOptions.KeyTransformer,
                OmitNullKeys = this._dictionarySerializationOptions.OmitNullKeys
            };
    }
}
```
should be added for backward compatibility too.

Reply:
I changed it by https://github.com/msgpack/msgpack-cli/pull/304/commits/0c609b2c6941e446364ae2d20a4be59b295563ee. Should I change it again like your snippet?

(I'm not sure how to use `#if` to avoid build errors in `[EditorBrowsable]`.)

```C#
private readonly DictionarySerializationOptions _dictionarySerializationOptions;

/// <summary>
/// Gets the dictionary(map) based serialization options.
/// </summary>
/// <value>
/// The <see cref="DictionarySerializationOptions"/> which stores dictionary(map) based serialization options. This value will not be <c>null</c>.
/// </value>
public DictionarySerializationOptions DictionarySerializationOptions
{
    get
    {
#if DEBUG
        Contract.Ensures( Contract.Result<DictionarySerializationOptions>() != null );
#endif // DEBUG
        return this._dictionarySerializationOptions;
    }
}

/// <summary>
/// Gets the dictionary(map) based serialization options.
/// </summary>
/// <value>
/// The <see cref="DictionarySerializationOptions"/> which stores dictionary(map) based serialization options. This value will not be <c>null</c>.
/// </value>
[Obsolete("Use DictionarySerializationOption instead.")]
#if XAMARIN && ( __ANDROID__ || __IOS__ )
[EditorBrowsable(EditorBrowsableState.Never)]
#endif
public DictionarySerializationOptions DictionarySerlaizationOptions
{
     get
     {
#if DEBUG
        Contract.Ensures( Contract.Result<DictionarySerializationOptions>() != null );
#endif // DEBUG
        return this._dictionarySerializationOptions;
    }
}
```

Reply:
Thank you!